scripts/copy-client-scripts.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cp -R node_modules/bootstrap/less/ src/client/stylesheets/bootstrap
```

Comment:
Don't think so. They're only called by prepublish, which is already cleaning and setting up directories first. This is only an issue if someone calls copy-client-scripts.sh directly, and if they do that, they should have a good reason.